opencsp/common/lib/render/View3d.py

```diff
@@ -518,7 +587,9 @@ def draw_xyz_text(self, xyz, text, style=rctxt.default()):  # An xyz is [x,y,z]
             )
             assert False
 
-    def draw_xyz(self, xyz, style: rcps.RenderControlPointSeq = None, label: str = None):  # An xyz is [x,y,z]
+    def draw_xyz(
+        self, xyz: tuple[list, list], style: rcps.RenderControlPointSeq = None, label: str = None
+    ):  # An xyz is [x,y,z]
         """Plots a single point, I think (BGB)."""

```

Comment:
Please add full doc string here.

Reply:
I kinda went overboard, but at least now this method has a full docstring!
https://github.com/sandialabs/OpenCSP/pull/142/commits/f3e689e644c0b61e562839aa594ddc18572c7f90